modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -0,0 +1,105 @@
+// Used in cicd/builds/running-entitled-builds.adoc
+
+[id="builds-running-entitled-builds-with-sharedsecret-objects_{context}"]
+= Running entitled builds with SharedSecret objects
+
+[role="_abstract"]
+Rather than copying your {op-system-base} subscription credentials and certificates into a builds' namespace as a Secret, you can access your credentials and certificates from a `Secret` object in one of the {product-title} system namespaces. To access this `Secret` object, you run entitled builds with CSI volume mount of a `SharedSecret` custom resource (CR) instance that references the `Secret` object.
+
+This procedure relies on the newly-introduced Shared Resources feature and the Insights Operator in {product-title}.
```

Comment:
@gabemontero Is "Shared Resources" another name for the "Shared Resources CSI Driver"?

Reply:
"Shared Resources" IMO based on context could be short hand for either the CSI Driver, the two new CRDs/API, or both the CSI Driver and the new CRDs/API we built (i.e. the "end to end solution"

In some places I use it as a short hand for "SharedSecrets and SharedConfigMaps".

Reply:
Updated. Thanks.